packages/cashier/src/pages/withdrawal/crypto-withdraw-receipt/crypto-withdraw-receipt.tsx

```diff
@@ -166,7 +165,6 @@ const CryptoWithdrawReceipt = observer(() => {
                     primary
                 />
             </div>
-            {isMobile() && isCryptocurrency(currency) ? <RecentTransaction /> : null}
```

Comment:
No need to render side notes separately in mobile mode inside child components as it will now be handled by `<PageContainer />`